brigade/templates/base.html

```diff
@@ -48,17 +46,17 @@
 
     </div>
 
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+      ga('create', 'UA-20825280-1', 'auto');
+      ga('require', 'linkid', 'linkid.js');
+      ga('send', 'pageview');
+    </script>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="{{ url_for('static', filename='js/application.js') }}"></script>
+    {% block js %}{% endblock %}
```

Comment:
We should add the scripts (except the Google Analytics) to `{% block js %}`, and in child templates with additional JavaScript, we should add a `{% block js %}`  and use `{{ super() }}` to inherit the parent scripts, followed by the additional JavaScript. This will allow us to override the parent JavaScript on a particular template in the future, if we ever need to do that.